abi.md

```diff
@@ -110,6 +110,51 @@ Passive bundles shall be lowered to Verilog packed structs.
 Reference types in ports shall be logically split out from aggregates and named
 as though "Aggregate Type Lowering" was used.
 
+### Remotely Instantiated Modules
+
+Remotely instantiated modules are lowered using the SystemVerilog
+`bind`{.verilog} statement.  All bind statements are placed in a separate file
+named `bindings_<circuit name>.sv`.
+
+For example, consider the following circuit `Foo`{.firrtl}` with remotely
+instantiated module `Bar`{.firrtl}:
+
+``` firrtl
+circuit Foo :
+  module Bar :
+    input a: UInt<1>
+
+  module Foo :
+    input b: UInt<1>
+
+    remoteinst bar of Bar
+    bar.a <= b
+```
+
+This will produce the following two SystemVerilog modules:
+
+``` verilog
+module Foo(
+  input b
+);
+endmodule
+
+module Bar(
+  input a
+);
+endmodule
+```
```

Comment:
I'm not sure it's necessary to mention in the spec, but these `Foo` and `Bar` are empty modules which have caused a number of errors in downstream tools. Though that's codegen internals, but it might be worth mentioning that these modules must be erased/inlined for correctness somewhere.

Reply:
Good point. I was attempting to keep the example compact by keeping the modules empty. However, it occurred to me that these _should_ be removed by optimizations. I can see if there's a way to make the examples in this PR or in #108 obviously block removal without making them overly long.